app/assets/stylesheets/component_guide/application.scss

```diff
@@ -136,6 +136,13 @@ $gem-guide-border-width: 1px;
   &.component-output {
     padding: 0;
   }
+
+  @include govuk-media-query($media-type: print) {
+    &[class*="-background"] {
+      color: $govuk-print-text-colour;
```

Comment:
Good change, makes sense, but why also set the `color` - `.component-guide-preview` elements shouldn't ever have any child elements or text other than components. I wouldn't want this rule to accidentally impact print styles for components in the guide so it looks like they print correctly but when used in applications actually print differently.

Reply:
I thought this was a good catch all, but you're right, it has the potential to cause problems. I've removed it. Other components that have inverse variants should already be handled in #4073.